starlight/templatetags/starlight_extras.py

```diff
@@ -8,4 +10,10 @@ def search(wanted_value, list_values):
     return any(element.skill == wanted_value for element in list_values)
 
 
+@register.filter(name='get_employees')
```

Comment:
This should be handled in the view for effective filtering purposes, not in a template tag.